src/Domain.LinnApps/Boms/BomVerificationHistoryService.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace Linn.Purchasing.Domain.LinnApps.Boms
+{
+    using System.Collections.Generic;
+    using Linn.Common.Authorisation;
+    using Linn.Common.Persistence;
+    using Linn.Purchasing.Domain.LinnApps.Exceptions;
+    using System;
+    using Linn.Purchasing.Domain.LinnApps.Parts;
+    using Linn.Common.Domain.Exceptions;
+    using Linn.Common.Proxy.LinnApps;
+
+    public class BomVerificationHistoryService : IBomVerificationHistoryService
+    {
+        private readonly IAuthorisationService authService;
```

Comment:
this looks unused. but it probably should be if you want to lock down creating these things

Reply:
Yeah, I'd asked Jacki about this but she didn't think it would really need any Authorisation. Keeping for now in case it's needed for the adjusting of the frequency but will most likely remove.